pilco/environments/environment.py

```diff
@@ -1,22 +1,34 @@
+from .continuous_cartpole import Cartpole
+from .continuous_mountaincar import Mountaincar
```

Comment:
I would put these in a subdirectory within `environments`, e.g. `environments.custom_gym`, because it currently looks like you're importing subclasses into the file of the superclass